zebrunner/appium
# remove func for making device unhealthy for error: the session ident

Changes made 
- remove function for making device Unhealthy for error : the session identified by ... is not known

Steps for QA 
- build appium with this update 
- run test with error from ticket https://github.com/zebrunner/appium/issues/398 
- you need to see that device is 'In Use'
